code/modules/mob/living/carbon/life.dm

```diff
@@ -730,21 +730,21 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	var/obj/item/organ/stomach/belly = getorganslot(ORGAN_SLOT_STOMACH)
 	if(!belly)
 		return FALSE
-	. = belly.reagents.has_reagent(reagent, amount, needs_metabolizing)
+	return belly.reagents.has_reagent(reagent, amount, needs_metabolizing)
 
-/mob/living/carbon/remove_reagent(reagent, amount=-1, safety)
-	if(amount == -1)
-		amount = get_reagent_amount(reagent)
+/mob/living/carbon/remove_reagent(reagent, custom_amount, safety)
+	if(!custom_amount)
```

Comment:
```suggestion
	if(isnull(custom_amount))
```
In case you want zero to be a valid amount. If not, disregard this suggestion.

Reply:
I see no reason that 0 is a valid amount to remove, this has no side benefit to have a removal of 0 be worth it.